tcp-native/src/main/java/org/ballerinalang/stdlib/tcp/endpoint/ClientActions.java

```diff
@@ -74,10 +74,10 @@ public static Object initEndpoint(Environment env, BObject client, BMap<BString,
             client.addNativeData(SocketConstants.SOCKET_SERVICE,
                     new SocketService(socketChannel, env.getRuntime(), callbackService, readTimeout));
         } catch (SocketException e) {
-            returnValue = SocketUtils.createSocketError("unable to bind the local socket port");
+            returnValue = SocketUtils.createSocketError("unable to bind the local tcp port");
         } catch (IOException e) {
-            log.error("Unable to initiate the client socket", e);
-            returnValue = SocketUtils.createSocketError("unable to initiate the socket");
+            log.error("Unable to initiate the client tcp", e);
+            returnValue = SocketUtils.createSocketError("unable to initiate the tcp");
```

Comment:
```suggestion
            returnValue = SocketUtils.createSocketError("unable to initiate the socket");
```
Please check the other locations as well where _TCP_ is not meaningful when used instead of the word, _Socket_.

Reply:
replaced tcp with socket